### PR TITLE
Added VIRUSBreakend dependencies

### PR DESCRIPTION
**Pull request by d-cameron**
Added Kraken2 and RepeatMasker dependencies

- GRIDS 2.10.0 is pre-release. Not sure what we want to do about this. Not merge until I formally release it, or just update common-tools when it does get released.

### CODE REVIEW

File: build/repeatmasker.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+sudo apt-get -y install perl-modules cpanminus python3-h5py
```
Comment (nedleitch) [inline build/repeatmasker.sh:1]: Can you move this to `cluster/images/build/repeatmasker.sh` please? Top-level directories should be added judiciously and I feel as though having it nearer the rest of the scripts will help make it clear what it's for.

Also maybe just some comments at the top to indicate what it's for OR an amendment to the `cluster/images/README.md` file so people know they need to run it when the OS/major dependencies change but not on a regular basis, would be useful.

**Review by nedleitch — CHANGES_REQUESTED** [review]

File: build/repeatmasker.sh (hunk 1 above)
Comment (d-cameron) [reply build/repeatmasker.sh]: Added to cluster/images/build subdirectory. This still keeps it seperate from the image building scripts, but doesn't introduce a new top-level directory

**Review by nedleitch — APPROVED** [review]
Looks good, thanks Daniel! I'll leave it open in case there's anything else you want to do but feel free to merge.

**Comment by pauldwolfe** [comment]
Closing this PR but leaving the branch in place.

**Pull request closed without merging**